packages/docusaurus-theme-classic/src/theme/LastUpdated/index.tsx

```diff
@@ -34,7 +34,7 @@ function LastUpdatedAtDate({
       values={{
         date: (
           <b>
-            <time dateTime={atDate.toISOString()}>
+            <time dateTime={atDate.toISOString()} itemProp="dateModified">
```

Comment:
Rather than using `itemProp` please can you move this to the JSON-LD Structured data for blog posts?

Reply:
@johnnyreilly That's actually the code I wanted to comment on 😅 

The problem is that this component is shared between docs and blog.
And docs don't use JSON LD atm but microdata.
So, not adding it there means `dateModified` structured data will not be applied to docs.

That's why I'm suggesting to add it to both. Eventually, we could use JSON LD for docs too later maybe?

Reply:
Sounds like a plan!

Reply:
@OzakIOne we still need to add `dateModified` in 2 places of `structuredDataUtils.ts` (alongside `datePublished`)

Reply:
I've added `dateModified` in `structuredDataUtils.ts` depending on if there is an `lastUpdatedAt` date it is returned in ISO format as requested by google and otherwise return an empty string seems to be the correct way [Rich Result Text Playground](https://search.google.com/test/rich-results/result/r%2Farticles?id=ftjLYZaOB-dboNv41HBzzQ)